### DISCUSSION TITLE
sync android to linux?

### DISCUSSION THREAD

**Discussion by otaolafranc** [remotely-save/remotely-save#787] [Q&A] [answered]
Hello,
i installed the remotely save pluging in the obsidian app in linux (appimage) and android, I entered the same credentials for webdav. but when I hit the 'check' I am getting 'the webdav server cannot be reached (possible to be any of adress/username/authtype errors) error invalid response 405 method not allowed.but the server link, username and passwords are the correct ones.... what I am missing?

**Comment by fyears** [comment]
do both devices not work?

**Reply by otaolafranc** [reply]
yes, I am getting the same error, where If I go to the link that I use in the 'server adresse' by copy/paste and then use the username and password to log (someones as used in obsidian plugging) i can connect without issues.

![image](https://github.com/user-attachments/assets/0545353f-0c8b-4c14-8011-147360db1aa8)

**Reply by fyears** [reply]
go to your webdav website, do you notice any folder or file been created?

**Reply by fyears** [reply]
the latest notice shows that your webdav server doesn't support some operations. maybe you need to adjust some operation permissions

**Reply by otaolafranc** [reply, marked as answer]
just solved!
I thought that the 'server adresse'  was the link to connect, but actually I should connect and go to files settings/ webDav and copy that link and use it for 'server adresse'
